frontend/src/components/activity/content/Material.vue

```diff
@@ -6,6 +6,7 @@
       :layout-mode="layoutMode"
       :material-item-collection="materialItemCollection"
       :disabled="disabled"
+      :disable-period-filter="true"
```

Comment:
```suggestion
      disable-period-filter
```
This is enough, as it coerces to true